index.html

```diff
@@ -6,11 +6,91 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <div class="container">
+
+        <div class="icon_img box">
+            <img src="https://www.bing.com/th?id=OIP.5wHPVNq-5T5Adqavx62ouwAAAA&w=150&h=150&c=8&rs=1&qlt=90&o=6&pid=3.1&rm=2"
+                alt="icon" width="100px">
+        </div>
+
+        <div class="bor">
+            <ul class="bor_li box">
+                <li class="">Home</li>
```

Comment:
I know this site is mainly for demoing grid, but remember these would be links "in real life"